src/util/blockchain.ts

```diff
@@ -40,11 +41,35 @@ export const CHAIN_IDS: ChainIds = {
   'eth': 2,
   'neo': 4,
   'bsc': 6,
-  'zil': 9,
+  'neo3': 14,
+  'zil': 18,
   'osmosis': 244,
   'terra': 245,
 };
 
+export const CHAIN_IDS_DEV: ChainIds = {
+  'eth': 350,
+  'neo': 5,
+  'neo3': 88,
+  'bsc': 79,
+  'zil': 111,
+};
+
+export const CHAIN_IDS_TEST: ChainIds = {
+  'eth': 2,
+  'neo': 5,
+  'neo3': 88,
+  'bsc': 79,
+  'zil': 111,
+};
+
+export const getChainIdFromBlockchain: SimpleMap<ChainIds> = {
```

Comment:
Hmm `getChainIdFromBlockchain` sounds a bit like a function name. Change to a more constant-sounding name (like `chainIdsByBlockchain`)